[conda] Fix comment replacement that crash conda build (#1127)
openfisca/openfisca-core: .conda/README.md

- Put the token in a CI environment variable named `ANACONDA_TOKEN`.

## Manual actions to test before CI

Everything is done by the CI but if you want to test it locally, here is how to do it.

Do the following in the project root folder:

- Auto-update `.conda/meta.yaml` with last infos from pypi by running:

>     - `python .github/get_pypi_info.py -p OPenfisca-core`

Reviewer: ```suggestion
    - `python .github/get_pypi_info.py -p OpenFisca-Core`

```

- Build package:

>     - `conda install -c anaconda conda-build anaconda-client`

Reviewer: ```suggestion
    - `conda install -c anaconda conda-build anaconda-client` (`conda-build` to build the package and [anaconda-client](https://github.com/Anaconda-Platform/anaconda-client) to push the package to anaconda.org)

```

>     - `conda build -c conda-forge .conda`

Reviewer: 👍 the explicit channel!

>  - Upload the package to Anaconda.org, but DON'T do it if you don't understand what you are doing:

Reviewer: > Do we know when we don't know? 😅 
```suggestion
 - Upload the package to Anaconda.org, but DON'T do it if you don't want to publish your locally built package as official openfisca-core library:

```
